File: backend/src/main/java/com/yinnohs/gamesprer/notification/domain/Notification.java
package com.yinnohs.gamesprer.notification.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
public class Notification {
    private String id;
    private String title;
    private String message;
    private String type;
    private String UserId;
    private String gameToFindTitle;
}

File: scraper/src/notifications/Notification.ts
export type NotificationType = "success" | "error";

export class Notification {
    public readonly title: string;
    public readonly message: string;
    public readonly type: NotificationType;
    public readonly userId: string;
    public readonly gameToFindTitle: string;

    constructor(
        title: string,
        message: string,
        type: NotificationType,
        userId: string,
        gameToFindTitle: string
    ) {
        this.title = title;
        this.message = message;
        this.type = type;
        this.userId = userId;
        this.gameToFindTitle = gameToFindTitle;
    }
}